client/component/pane.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Graph from './graph';
+
+const data = [
+  {
+    date: '1',
+    weight: 200
+  },
+  {
+    date: '2',
+    weight: 198.1
+  },
+  {
+    date: '3',
+    weight: 200
+  },
+  {
+    date: '4',
+    weight: 198
+  },
+  {
+    date: '5',
+    weight: 204
+  },
+  {
+    date: '6',
+    weight: 192
+  },
+  {
+    date: '7',
+    weight: 190
+  }
+];
+
+export default class Pane extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+
+    let body;
+    if (this.props.type === 'chart') {
+      if (data) {
+        body = <Graph data={data} />;
+      } else {
+        body = <div className="pane-body">
+          No data found!
+        </div>;
+      }
+
+    } else {
+      body =
+      <div className="pane-body">
+        {this.props.body}
+      </div>;
```

Comment:
FYI, there's a special `props.children` (if you choose to use it) that can be used to nest other components inside custom components: https://reactjs.org/docs/composition-vs-inheritance.html#containment